src/server/metricsServer.js

```diff
@@ -20,8 +20,10 @@ import { register as metricsRegister, collectDefaultMetrics } from 'prom-client'
 
 import logging from './utils/logging/serverMiddleware';
 import healthCheck from './middleware/healthCheck';
+import { cacheSizeCollector } from './metrics/intl-cache';
 
 collectDefaultMetrics();
+metricsRegister.registerCollector(cacheSizeCollector);
```

Comment:
@infoxicator the register ensures that the cacheSizeCollector is called in the same manor default metrics are collected